plugins/signature_provider_plugin/signature_provider_plugin.cpp

```diff
@@ -42,6 +42,9 @@ class signature_provider_plugin_impl {
       std::tuple<std::string, std::string, std::string> parse_spec(const std::string& spec) const {
          auto delim = spec.find("=");
          EOS_ASSERT(delim != std::string::npos, chain::plugin_config_exception, "Missing \"=\" in the key spec pair");
+         // public_key can be base64 encoded with trailing `=`
+         while( spec.size() > delim+1 && spec[delim+1] == '=' )
+            ++delim;
          auto pub_key_str = spec.substr(0, delim);
          auto spec_str = spec.substr(delim + 1);
```

Comment:
Will this be out of range if `=` is the last char in `spec` and `delim` points to `=`?